lib/hydra/works/models/concerns/collection_behavior.rb

```diff
@@ -21,6 +21,15 @@ module CollectionBehavior
       type [Hydra::PCDM::Vocab::PCDMTerms.Collection, Vocab::WorksTerms.Collection]
     end
 
+    module ClassMethods
+      def type_validator
+        Hydra::PCDM::Validators::CompositeValidator.new(
```

Comment:
Can you just add the new validator to the `validators` array?  Maybe after making sure it's not nil?